pandas/core/indexes/multi.py

```diff
@@ -1126,7 +1126,7 @@ def _to_safe_for_reshape(self):
         """ convert to object if we are a categorical """
         return self.set_levels([i._to_safe_for_reshape() for i in self.levels])
 
-    def to_frame(self, index=True):
+    def to_frame(self, index=True, names=None):
```

Comment:
This should be `name`, singular right? Changing the signature of a method in a subclass is generally a bad idea.

Reply:
oh yes I c. though we do accept name/names in many methods we should just not add to the problem (IIRC we are changing that signature in another issue)